controller/pagecontroller.php

```diff
@@ -62,15 +63,21 @@ public function index() {
         // check if note exists
         try {
             $this->notesService->get($lastViewedNote, $this->userId);
+            $errorMessage=null;
         } catch(NoteDoesNotExistException $ex) {
             $lastViewedNote = 0;
+            $errorMessage="The last viewed note cannot be accessed. ".$ex->getMessage();
+        } catch(DecryptionFailedException $ex){
+            $lastViewedNote = 0;
+            $errorMessage=$ex->getMessage();
         }
```

Comment:
Can we merge all exceptions to a single `catch(\Exception)` here? Then we are prepared if a specific exception type is missing.

Reply:
Sure. I think the rationale for doing it this way was to append messages to the first exception which often returns with a blank message, but I can just prefix to every message.